cmd/adapter-rest/startcmd/start.go

```diff
@@ -628,13 +638,36 @@ func addRPHandlers(
 	return nil
 }
 
+func issueGovernanceCredential(governanceVCSURL string, rootCAs *x509.CertPool, store storage.Provider) error {
```

Comment:
Governance credentials are issued per RP and per Issuer, not per adapters.

The issuance of these credentials should be plugged in to the respective "create tenants" endpoints in the operations.

Also, the credential subject of these credentials should reference the public DIDs of the rps/issuers.

Reply:
please reference where is the create tenants endpoints

Reply:
@fqutishat 

* issuer: https://github.com/trustbloc/edge-adapter/blob/d1e8c1705d913b72a15c91c842199c3ed123802c/pkg/restapi/issuer/operation/operations.go#L205
* rp: https://github.com/trustbloc/edge-adapter/blob/f41c373b400728be416f4f57a9b1faa14e1ca325/pkg/restapi/rp/operation/operations.go#L1105

Reply:
@llorllale I will move the logic to  "create tenants" endpoints and add public dids

Reply:
done